config/tests/samples/create/samples_test.go

```diff
@@ -64,6 +64,8 @@ var testDisabledList = map[string]bool{
 	// The resources below have special test requirements
 	"computeinterconnectattachment": true,
 	"firestoreindex":                true,
+	"edgenetworknetwork":            true,
```

Comment:
I am missing the context. Why are these disabled?

Reply:
Edge Network API is a special case among google cloud APIs in that it must connect to a special hardware deployment, an "edge zone", to create resources. (See: https://cloud.google.com/distributed-cloud/edge/latest/docs/how-it-works) Internal to Google, we can provision edge zones for testing purposes, but they can't be exposed here. Therefore the only possible integration test for Edge Network API is with a mock.

In magic-modules, it was decided to disable the integration tests for edgenetwork because they have no mocking framework. (see comments here: https://github.com/GoogleCloudPlatform/magic-modules/pull/8905)